pkg/apis/config/types.go

```diff
@@ -58,7 +55,7 @@ type NodeResourcesAllocatableArgs struct {
 	// An example resource set might include "cpu" (millicores) and "memory" (bytes)
 	// with weights of 1<<20 and 1 respectfully. That would mean 1 MiB has equivalent
 	// weight as 1 millicore.
-	Resources []schedulerconfig.ResourceSpec `json:"resources,omitempty"`
+	Resources []unversioned.ResourceSpec `json:"resources,omitempty"`
```

Comment:
`unversioned` is a little strange. Could we change to `schedconfig` ?

Reply:
done.